docker/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Docker file for EP2017 python container
+FROM python:2.7.9
+
+ENV PYTHONUNBUFFERED 1
+ENV PIP_NO_CACHE_DIR 0
+
+RUN mkdir /website /website/data /website/data/site
+WORKDIR /website
+
+# Upgrade System:
+RUN apt-get update && apt-get -y upgrade && apt-get -y autoremove && apt-get install -qq build-essential libssl-dev libxrender-dev wget git wkhtmltopdf libpango1.0-0 libcairo2 libpq-dev && rm -rf /var/lib/apt/lists/*
```

Comment:
Running `apt-get upgrade` is not recommended in Dockerfiles, see subsection *apt-get* in https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run